microsite/data/plugins/aws-app-development.yaml

```
@@ -0,0 +1,10 @@
+---
+title: App Development for Backstage.io on AWS
+author: Amazon Web Services
+authorUrl: https://aws.amazon.com/
+category: Infrastructure
+description: Create and develop applications on AWS using Backstage.
```

Trying to make this a bit more concise but feel free to tweak it further.
 
```suggestion
description: Create and manage AWS Apps within Backstage```